modules/localgov_campaigns_paragraphs/config/install/media.type.document.yml

```diff
@@ -0,0 +1,13 @@
+langcode: en
+status: true
+dependencies: {  }
+id: document
+label: Document
+description: 'An uploaded file or document, such as a PDF.'
+source: file
+queue_thumbnail_downloads: false
+new_revision: true
+source_configuration:
+  source_field: field_media_document
+field_map:
+  name: name
```

Comment:
This media definiation is already supplied by the localgov_media module (https://github.com/localgovdrupal/localgov_core/blob/master/modules/localgov_media/config/optional/media.type.document.yml) and so is not needed here.

Reply:
Removed